CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Changelog
+
+The format of this document is inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and the project follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- This is a comment, you won't see it when GitHub renders the Markdown file.
+
+When releasing a new version:
+
+1. Remove any empty section (those with `_None._`)
+2. Update the `## Unreleased` header to `## [<version_number>](https://github.com/wordpress-mobile/Automattic-Tracks-iOS/releases/tag/<version_number>)`
```

Comment:
Nice catch updating this URL on every PR you make to add those CHANGELOGs in our lib repos each time… it could be so easy to miss when doing a quick copy/paste, so kudos for the attention to detail 🙂 👍

Reply:
Thanks! 😄

Reply:
Actually... While working on the [`tumblr/Kanvas-iOS` one](https://github.com/tumblr/kanvas-ios/pull/148), I realized I didn't update the org part of this GitHub URL 😓

Reply:
https://github.com/Automattic/Automattic-Tracks-iOS/pull/235

Reply:
And somehow we both missed that, even after I commented on that exact line about that URL saying it was the right one 🤦 
We both need to update our coffee blend to stronger beans it seems 😄  ☕ ☕ ☕